lodash.js

```diff
@@ -14865,6 +14875,8 @@
       var variable = hasOwnProperty.call(options, 'variable') && options.variable;
       if (!variable) {
         source = 'with (obj) {\n' + source + '\n}\n';
+      } else if (reForbiddenIdentifierChars.test(variable)) {
```

Comment:
I noticed there is no check that `variable` is `typeof 'string'`

Reply:
`RegExp.prototype.test` coerces its argument to a string before making matches.